vllm/model_executor/layers/rotary_embedding.py

```diff
@@ -25,6 +25,9 @@
 import math
 from typing import Any, Dict, List, Optional, Tuple, Union
 
+import numba
+from numba import jit
```

Comment:
Just for a heads up: Numba is a dependency only for the cuda/rocm backends. We need to move Numba from `requirements/cuda.txt` to `requirements/common.txt`.

Reply:
Thanks for pointing out this, I have moved the numba dependency to `requirements/common.txt`

Reply:
> When you change dependencies in the .txt files I believe you also need to recompile those into the .in files

I think only `requirements/test.txt` is generated by `requirements/test.in`, other requirements files use `.txt` only. And `requirements/test.in` already contains the `numba` dependency, so this PR didn't touch any `.in` file.